drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/report.rb

```diff
@@ -25,6 +25,7 @@ class Report < HudReports::ReportInstance
     has_many :inventories
 
     after_initialize :filter
+    alias_attribute :result_cache, :build_for_questions
 
```

Comment:
add a `diet` scope.
invalidate existing result_cache if you call run and save
show details should be have like modal, no shortening.